src/app/datagrid/selection-single/selection-single.html

```diff
@@ -40,7 +40,7 @@ <h3>Client side, no trackBy</h3>
   <clr-dg-column>Creation date</clr-dg-column>
   <clr-dg-column>Favorite color</clr-dg-column>
 
-  <clr-dg-row *clrDgItems="let user of users" [clrDgItem]="user">
+  <clr-dg-row *clrDgItems="let user of users" [clrDgItem]="user" [clrDgRowAriaLabel]="'Select row for ' + user.name">
```

Comment:
This updates the demos on the angular docs application to create unique user names.